device/RPI2.conf

```diff
@@ -1,13 +1,14 @@
 # https://www.raspberrypi.org/products/raspberry-pi-2-model-b/
 
 export MAKE_ARGS_DEV="
-CROSS_BINUTILS_PREFIX=/usr/local/arm-gnueabi-freebsd11.2/bin
```

Comment:
this should use `CROSS_BINUTILS_PREFIX=/usr/local/arm-gnueabi-freebsd${PRODUCT_ABI}/bin`

because arm-none-eabi will cause many problem when compiling kernel, especially with cloudabi and [armelf_fbsd](https://github.com/opnsense/src/blob/master/sys/conf/kern.mk#L280)
if you found any workaround instead of manually editing kern.mk to make it useable with none-eabi, it would be great.

this applies to all armv7h

Reply:
Done, thanks

Reply:
Actually 11.2 corresponds to SRCABI, not PRODUCT_ABI

Reply:
Whoops, I meant SRCREVISION

I'll commit a fix to access that from the device config.

Reply:
316eb45bd7a

Reply:
resolved comment wasn't resolved